src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -496,6 +522,8 @@ export default {
 				return t('spreed', 'You have no unread mentions.')
 			case 'unread':
 				return t('spreed', 'You have no unread messages.')
+			case 'archived':
+				return t('spreed', 'You have no archived conversations.')
```

Comment:
Filter traces are seen in:
* Highlighted filter button
* Clear filter button
![image](https://github.com/user-attachments/assets/7827fc05-3037-4151-9eef-359e4221e18d)

Reply:
Maybe it's better to implement separately from filters then